Changelog

```diff
@@ -1,3 +1,67 @@
+Version 3.1, 2019-09-XX
```

Comment:
Sorry, I only now noticed we're missing some issues that had no milestone:
#1789
#1782
#1707
#1663

Reply:
Oups, some of them were so small that I was not sure, what to write (TiQR) and some obviously missed the milestone tag (I was going via the issues in the milestone)

But will do.